tools/slo-generator/Makefile

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/make
+# WARN: gmake syntax
+########################################################
+# Makefile for $(NAME)
+#
+# useful targets:
+#	make check -- manifest checks
+#	make clean -- clean distutils
+#	make coverage_report -- code coverage report
+#	make flake8 -- flake8 checks
+#	make pylint -- source code checks
+#	make rpm -- build RPM package
+#	make sdist -- build python source distribution
+#	make systest -- runs the system tests
+#	make tests -- run all of the tests
+#	make unittest -- runs the unit tests
+#
+# Notes:
+# 1) flake8 is a wrapper around pep8, pyflakes, and McCabe.
+########################################################
+# variable section
+
+NAME = "slo_generator"
+
+PIP=pip
```

Comment:
curious why this isn't `pip3` ?

Reply:
assuming that the `pip` points to either Python 2's `pip` or `pip3` if the environment is configured right. I've run the Makefile steps from a virtualenv everytime and `pip` is aliased to `pip3` there.